os/arch/Kconfig.chip

```diff
@@ -735,6 +740,15 @@ config RAM_SIZE
 		does not execute out of RAM but from FLASH, then you may designate
 		any block of RAM as "primary."
 
+config RAM_DDR_START
+	hex "DDR RAM start address"
+	default 0x0
```

Comment:
May be you can add some description in --help-- for more clarity. 
Thanks, Pradeep

Reply:
it will be changed as your guidance.

BR, HJ Lee.